decidim-conferences/app/helpers/decidim/conferences/conference_helper.rb

```diff
@@ -4,13 +4,73 @@ module Decidim
   # A Helper to render conferences.
   module Conferences
     module ConferenceHelper
+      include PaginateHelper
+
       # Renders the dates of a conference
       #
       def render_date(conference)
         return l(conference.start_date, format: :decidim_with_month_name_short) if conference.start_date == conference.end_date
 
         "#{l(conference.start_date, format: :decidim_with_month_name_short)} - #{l(conference.end_date, format: :decidim_with_month_name_short)}"
       end
+
+      # Items to display in the navigation of a conference
+      #
+      def conference_nav_items
```

Comment:
I find it hard to navigate the conference when I go e.g. to the "page" section but maybe this will be fixed when the header part is implemented.

I am just trying to imagine that the breadcrumbs and the mega menu would actually work and I would still find it cumbersome that e.g. if I go to see an information page about the conference or the program page, I cannot easily navigate to the other sections without opening the mega menu or going back to the conference main page and selecting the other link there.

Reply:
@ahukkanen should you mention @decidim/product for this kind of issues? This is  not something we can address in this PR were only the approved redesign is being implemented.

Reply:
@ferblape I think I've mentioned it a couple of times in different occasions before (such as the first redesign meeting where this was first published) but yes I understand it cannot be addressed in this PR.

I just wanted to bring that up.

Reply:
Just to chime in, this has come up too while we were reviewing with @decidim/product: it's really difficult to imagine the general navigation and look and feel when you don't have the breadcrumb (as its an integral part of the new design). You're always on "Imaginem el nou ecobarri de Ca n' Ametller".

You always need to play with the URL or use the browsers' "Go back one page" functionality to check things in the new design.

Reply:
It was mentioned in the last PX meeting that this is being implemented right now, so I will mark this resolved regarding this PR.